src/api/app/models/report.rb

```diff
@@ -21,7 +21,7 @@ class Report < ApplicationRecord
   scope :without_decision, -> { where(decision: nil) }
 
   def reason
-    return category.humanize if category != 'other'
+    return category.humanize if category.present? && (category != 'other')
```

Comment:
When testing in `irb` with `nil.present?` I got
```
undefined method `present?' for nil:NilClass
```

So this probably will not improve handling of nil.
`nil.nil?` is defined, though.